addons/l10n_sa/data/account_tax_group.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <!-- Account Tax Group -->
+        <record id="sa_tax_group_taxes_15" model="account.tax.group">
+            <field name="name">Taxes 15%</field>
+            <field name="property_tax_payable_account_id" ref="sa_account_202003"/>
+            <field name="property_tax_receivable_account_id" ref="sa_account_100103"/>
+        </record>
+        <record id="sa_tax_group_taxes_other" model="account.tax.group">
+            <field name="name">Other Taxes</field>
+        </record>
+        <record id="sa_tax_group_taxes_withholding" model="account.tax.group">
+            <field name="name">Withholding Tax</field>
+        </record>
```

Comment:
I'm assuming those groups are only for Saudi Arabia ; we should give them a country_id then, so that they are not proposed on taxes from other countries.

Reply:
I was unaware of that new feature, i added them